app/scripts/controllers/exportImport.js

```diff
@@ -126,6 +126,12 @@ angular.module('openhimConsoleApp')
       var exportData = angular.copy( $scope.selectedExports );
       var textFile = null;
 
+      // OHM-382: warn when exporting key
```

Comment:
There's no need for this comment. The related ticket can be found from the Git history if required.